spec/models/prosecution_case_summary_spec.rb

```diff
@@ -11,4 +11,10 @@
   subject { prosecution_case_summary }
 
   it_has_behaviour 'conforming to valid schema'
+
+  describe 'hearings_builder' do
```

Comment:
I think this could be validated against the schema, similar to [these](https://github.com/ministryofjustice/hmcts-common-platform-mock-api/blob/67d92d4dfe41a490b46db920186ae4803fd29555/spec/models/court_application_party_counsel_spec.rb#L23-L30).

That would avoid us needing to define the hearing on the `prosecution_case` factory, since it is optional, and then we end up validating both cases